lib/node_modules/@stdlib/blas/base/dsyr/docs/repl.txt

```diff
@@ -46,11 +46,25 @@
 
     Examples
     --------
+    // Standard usage:
     > var x = new {{alias:@stdlib/array/float64}}( [ 1.0, 1.0 ] );
     > var A = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 0.0, 2.0 ] );
```

Comment:
We should update these examples such that `A` is actually a 2x2 symmetric matrix, rather than being assymetric.